src/Apps/Consign/Routes/SubmissionFlow/Mutations/UpdateConsignSubmissionMutation.ts

```diff
@@ -22,7 +23,7 @@ export const updateConsignSubmissionMutation = (
         }
       `,
       variables: {
-        input,
+        input: omitBy(input, isEmpty),
```

Comment:
@MounirDhahri was the decision to remove this in the end, suggestions how to resolve the issue?

Reply:
The issue in Eigen was that we were sending always all fields - this is fine in a flow that is uni-direction. It does however break if the user goes back and fires the mutation again, which can possibly reset already updated values. See this https://github.com/artsy/eigen/pull/10140#discussion_r1582833282